src/dotnet/Program.cs

```diff
@@ -113,6 +113,16 @@ private static int ProcessArgs(string[] args)
 
             var appArgs = (lastArg + 1) >= args.Length ? Enumerable.Empty<string>() : args.Skip(lastArg + 1).ToArray();
 
+            if (verbose.HasValue)
+            {
+                Environment.SetEnvironmentVariable(CommandContext.Variables.Verbose, verbose.ToString());
+            }
+            //This process should inherit AnsiPassThrough or use the default of false.
+            CommandContext.InitializeAnsiPassThroughFromEnvironment();
```

Comment:
How does this affect the logical flow of the program?

It reads the environment variable but how does that affect subsequent code which runs? I may be missing something here.